mailpoet/mailpoet
# Extend the pending approval message on settings/premium [MAILPOET-4633]

## Description

- This PR augments pending approval messages to include a link to recheck the status of the key and update 
- It fixes a bug that prevented the pending approval state to show up in case the status was already updated via clicking on the `Verify` button on the premium tab multiple times.

## Code review notes

The messages may get updated, per what @NeosinneR thinks, but the code should remain intact.

## QA notes
The pending approval message is shown in the following places:

- On the preview modal (opened from the email edit page) once `Send to email` is selected
- On `Send` page of email editor
- On the `Premium` tab in settings page

You can change the following data in DB to set your plugin in pending for approval state:
` {prefix}_mailpoet_settings -> key = mta -> set is_approved to 0`


## Linked tickets

[MAILPOET-4633]

## After-merge notes

Confirm with @samnajian or @NeosinneR for updated translation strings before merging.

[MAILPOET-4633]: https://mailpoet.atlassian.net/browse/MAILPOET-4633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
